src/android/InAppBrowser.java

```diff
@@ -1111,6 +1119,7 @@ public void postMessage(String data) {
                 if (openWindowHidden && dialog != null) {
                     dialog.hide();
                 }
+                dialog.getWindow().setBackgroundDrawableResource(android.R.color.transparent);
```

Comment:
does this need to be moved into that if statement check on the line above, just that existing code thinks it can be null?

Reply:
TBH I'm not sure why the above is wrapped in an if not null block because line 1108 also uses it without checking. Also on top of that the dialog is created in this scope at the top. My only thought is threading could be deleting this maybe? If so this is poorly handled all together.

Reply:
Yeah you are right, its doing it both ways and making it inside that method. Maybe they just copied and pasted it from somewhere else.